src/server/acl/acl_family.cc

```diff
@@ -152,25 +152,27 @@ void AclFamily::EvictOpenConnectionsOnAllProactorsWithRegistry(
   }
 }
 
+// Currently, Dragonfly supports only deleting one ACL user at a time.
+// If the deletion is successful, return 1, otherwise 0.
 void AclFamily::DelUser(CmdArgList args, ConnectionContext* cntx) {
   std::string_view username = facade::ToSV(args[0]);
   if (username == "default") {
-    cntx->SendError("The'default' user cannot be removed");
+    cntx->SendError("The 'default' user cannot be removed");
     return;
   }
 
+  long deleted = 1;
   auto& registry = *registry_;
   if (!registry.RemoveUser(username)) {
-    cntx->SendError(absl::StrCat("User ", username, " does not exist"));
-    return;
+    deleted = 0;
```

Comment:
Can you plz revert and just call `cntx->SendLong(0);` here? I will take care of deleting multiple users :)

Reply:
Yeah, I kind of feel like doing this... as you said, keep your original logic and avoid unnecessary calls to `EvictOpenConnectionsOnAllProactors`. 😃